scalableminds/webknossos
# Remove deprecation warnings

### Steps to test:
- run `./clean`
- run `yarn start`
- there should not be any deprecation warnings of play

### Issues:
- fixes #3190
------
